mapbox/utils.py

```diff
@@ -0,0 +1,30 @@
+from collections import Mapping, Sequence
+
+def normalize_geojson_featurecollection(obj):
+    """Takes a geojson-like mapping representing
+    geometry, Feature or FeatureCollection (or a sequence of such objects)
+    and returns a FeatureCollection-like dict
+    """
+    if not isinstance(obj, Sequence):
+        obj = [obj]
+
+    features = []
+    for x in obj:
+        if not isinstance(x, Mapping) or 'type' not in x:
+            raise ValueError(
+                "Expecting a geojson-like mapping or sequence of them")
+
+        if x['type'] == 'FeatureCollection':
```

Comment:
I'd like to see `'features' in x and` in this expression, would intercept possible `KeyError` and lead to the intended `ValueError`.